Documentation/operations/performance/benchmark.rst

```diff
@@ -18,6 +18,9 @@ All tests are performed between containers running on two different bare metal
 nodes connected back-to-back by a 100Gbit/s network interface. Upon popular
 request we have included performance numbers for Calico for comparison.
 
+Also, you can watch Thomas Graf, Co-founder of Cilium, dive deep into this chapter 
```

Comment:
Not sure what's best, “dive deep” or “deep-dive”?

Reply:
I think "dive deep" reads better for the sentence.